config/manifests/bases/runtime-component.clusterserviceversion.yaml

```diff
@@ -15,290 +15,31 @@ spec:
   customresourcedefinitions:
     owned:
     - description: RuntimeComponent is the Schema for the runtimecomponents API.
```

Comment:
Wondering why these are being removed by the make commands on your machine. Are you using operator-sdk 1.6.4? Did you run `make generate && make manifests && make bundle`?

Reply:
Discussed this with Melissa, and this is duplicate content (resulted from v1beta1 to v1beta2 change), so removing is the right step.